### PR TITLE
feat(coercion): add coercion math

### PR DESCRIPTION
**Pull request by wenzi0github**
When rounding data, it is recommended to use the method under "math" for operation of pure numeric data, and "parseInt" for string type numbers, which is more semantic and more in line with the definition of function parameters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3205,6 +3205,20 @@ Other Style Guides
...
+
+    ```javascript
+    const num = 12.6;
+    const strNum = '12.6';
+
+    const val = Math.floor(num); // 12
+
+    const val = Math.ceil(num); // 13
+
+    const val = parseInt(strNum, 10); // 12
```
Comment (ljharb) [inline README.md]:
```suggestion
    const val = Number.parseInt(strNum, 10); // 12
```

Comment (wenzi0github) [reply README.md]: yes, Number.parseInt seems more obvious contrast.

**Comment by wenzi0github** [comment]
@ljharb this pull could be merged? thank you!

**Comment by wenzi0github** [comment]
difference `Math.floor` and `Number.parseInt`, Math always manage number type, parseInt always mange string type to number.

**Pull request closed without merging**